index.html

```diff
@@ -7,33 +7,42 @@
     <meta name="author" content="Daria Pavlyuk" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Daria Pavlyuk Portfolio</title>
+    <link rel  = "preconnect" href ="https://fonts.googleapis.com">
+    <link href ="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <link rel ="stylesheet" href = "css/index.css">
+    <link rel ="stylesheet" href ="css/media.css">
   </head>
   <body>
-    <!-- <img src="IMG_5301.jpeg" alt="Personal Photo"> -->
-    <h1 style="color: rgb(130, 207, 238)">Daria Pavlyuk</h1>
+    <ul>
+      <li><a href="#About">About</a></li>
+      <li><a href="#Experience">Experience</a></li>
+      <li><a href="#Connect">Connect</a></li>
+    </ul>
+    <div class = "wrapper">
+    <h1>Daria Pavlyuk</h1>
     <section id="About">
-      <h2 style="color: rgb(130, 207, 238)">About:</h2>
+      <h2>About:</h2>
       <p>
         My name is Daria Pavlyuk. I am a QA Engineer. I <em>enjoy</em> testing
         Web Apps. I am a student at CTD. Coding is <strong>FUN </strong>!
       </p>
     </section>
     <section id="Experience">
-      <h2 style="color: rgb(130, 207, 238)">Experience:</h2>
+      <h2>Experience:</h2>
       <ul>
```

Comment:
this is a list-item and needs to be styled to be distinct. You can add this

`<ul class="experience"></ul>`